### PR TITLE
Adding a check for the machines with the build, windows & x64 labels 

### LINKED ISSUE #2765: Nagios - Add Additional Jenkins Check Label Job To Nagios Server Group For ubuntu & fpm labels
Currently the nagios server has check_label jobs for various jenkins options such as

Check Label- arm/ci.role.test
Check Label- build/linux/s390x
Check Label- macos10.14/build/mac/x64
Check Label- wix 

A further check is required to be added for the machines with the build, windows & x64 labels

Please update the readme.md file in the ansible/playbooks/nagios/README.md with a new section detailing how to add an additional version of these checks.

### PR DESCRIPTION
**Pull request by lumuchris256**
Added a check for the machines with the build, windows & x64 labels  
 updated the readme.md file in the ansible/playbooks/nagios/README.md with a new section detailing how to add an additional version of these checks.
fixes #2765 

cc @steelhead31 @smlambert @sxa

### CODE REVIEW

File: ansible/playbooks/nagios/README.md [hunk 1]
```diff
@@ -62,3 +62,44 @@ The Encrypted File Can Have Its Password Changed With The Following command
...
+	    command_line  $USER1$/check_build_windows_x64 "$ARG1$" $ARG2$ $ARG3$
+
+	}
+```
+*  Amend the machine specific config file, e.g ( /usr/local/nagios/etc/objects/localhost.cfg ) to include the entry for the new label check.
+
+```bash
+	define service{
+		 use                             local-service
+        host_name                       Nagios_Server
```
Comment (karianna) [inline ansible/playbooks/nagios/README.md:98]: Not sure the indenting is right here.

File: ansible/playbooks/nagios/README.md [hunk 2]
```diff
@@ -62,3 +62,44 @@ The Encrypted File Can Have Its Password Changed With The Following command
...
+```bash
+nano commands.cfg
+```
+* add `check_build_windows_x64` command definition
+
+```bash
+define command{
+	    command_name	check_label_build_windows_x64
+	    command_line  $USER1$/check_build_windows_x64 "$ARG1$" $ARG2$ $ARG3$
+
```
Comment (karianna) [inline ansible/playbooks/nagios/README.md:90]: extra whitespace

**Review by karianna — CHANGES_REQUESTED** [review]

File: ansible/playbooks/nagios/README.md (hunk 2 above)
Comment (lumuchris256) [reply ansible/playbooks/nagios/README.md]: committed the requested changes

File: ansible/playbooks/nagios/README.md (hunk 1 above)
Comment (lumuchris256) [reply ansible/playbooks/nagios/README.md:97]: corrected

File: ansible/playbooks/nagios/README.md [hunk 3]
```diff
@@ -62,3 +62,42 @@ The Encrypted File Can Have Its Password Changed With The Following command
 Based off the [installation guide](https://support.nagios.com/kb/article/nagios-core-installing-nagios-core-from-source-96.html):
 And Off This [GitRepo](https://github.com/Willsparker/AnsibleBoilerPlates/tree/main/Nagios) :
 For some useful tips for working with vault files see [here](https://docs.ansible.com/ansible/latest/user_guide/vault.html)
+
+## How to Add Additional Jenkins Check Label Job To Nagios Server Group For windows ##
```
Comment (gdams) [inline ansible/playbooks/nagios/README.md:66]:
```suggestion
## How to add additional Jenkins Check Label Job To Nagios server group For Windows ##
```

**Comment by gdams** [comment]
1 nit but LGTM other than that

**Comment by lumuchris256** [comment]
> 1 nit but LGTM other than that

worked on

File: ansible/playbooks/nagios/README.md [hunk 4]
```diff
@@ -62,3 +62,42 @@ The Encrypted File Can Have Its Password Changed With The Following command
...
+nano commands.cfg
+```
+* add `check_build_windows_x64` command definition
+
+```bash
+define command{
+	    command_name	check_label_build_windows_x64
+	    command_line  $USER1$/check_build_windows_x64 "$ARG1$" $ARG2$ $ARG3$
+	}
+```
```
Comment (steelhead31) [inline ansible/playbooks/nagios/README.md:91]: This section isnt required, as check\_label already exists as a command, we dont need to create a duplicate

File: ansible/playbooks/nagios/README.md [hunk 5]
```diff
@@ -62,3 +62,42 @@ The Encrypted File Can Have Its Password Changed With The Following command
...
+*  Amend the machine specific config file, e.g ( /usr/local/nagios/etc/objects/localhost.cfg ) to include the entry for the new label check.
```
Comment (steelhead31) [inline ansible/playbooks/nagios/README.md:92]: Amend the machine specific config file is not relevant for adding a check run by the nagios server itself, the only file to be changed is local host,  Id change the words "machine specific" for "nagios server"

**Review by steelhead31 — CHANGES_REQUESTED** [review]
Some changes required....  as this is an instance of a new check, this section isnt needed

**Review by steelhead31 — APPROVED** [review]
LGTM, I've used this to add the check for windows hosts, which can be seen in production nagios server now.

**Comment by sxa** [comment]
@karianna Are you ok with this one now?

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**